packages/core/src/app/address/StaticAddress.scss

```diff
@@ -1,5 +1,18 @@
 .checkout-address--static {
+    $icon-size: 1.85rem;
+
+    position: relative;
+
     .address-entry {
         margin: 0;
+        padding-right: $icon-size;
```

Comment:
I undertand you want to make space for the icon.
However, this class also applies to the condition when there is no icon.
Could you please create a new CSS class for PP icon so that it won't impact existing UI?

Reply:
yes, you are right, it will be better, will do

Reply:
done